E/Exodus/build_tarballs.jl

```diff
@@ -92,7 +104,7 @@ products = [
 dependencies = [
     Dependency(PackageSpec(name="Fmt_jll", uuid="5dc1e892-f187-50dd-85f3-7dff85c47fc5"))
     # Updating to a newer HDF5 version is likely possible without problems but requires rebuilding this package
-    Dependency(PackageSpec(name="HDF5_jll", uuid="0234f1f7-429e-5d53-9886-15a909be8d59"); compat="~1.12")
+    Dependency(PackageSpec(name="HDF5_jll", uuid="0234f1f7-429e-5d53-9886-15a909be8d59"); compat="~1.14")
```

Comment:
You can't change a compat bound without changing the version number of the package. But I don't understand what's the problem. And in any case compat bounds have to be fixed first in the registry, not here.

Reply:
I bumped the version number to 8.19.1 due to the compat bound you mentioned.

What was happening was NetCDF_jll (sorry I wrote my initial request incorrectly) was causing an error during the audit on certain os/ABI combinations. It was looking for libmbedtls.so.7.0 but only libmbedtls.so.13.0 was found by the auditer. The compat change seemed to fix it. It's the compat version for NetCDF_jll in HDF5_jll.